### DISCUSSION TITLE
@solana/spl-token version mismatch

### DISCUSSION THREAD

**Discussion by niravvaghani** [metaplex-foundation/js-deprecated#200] [General]
HI
In my current project, @solana/spl-token@0.2.0 already installed. Now I have added metaplex/js package. but in this package, you have used 0.1.8 version of @solana/spl-token. Now it is conflicting. can you not change version of spltoken package to latest version ?? 

I am not able to use 
import { getOrCreateAssociatedTokenAccount, transfer } from '@solana/spl-token';
as in 0.1.8 version above funtion are not there.

**Comment by kfnx** [comment]
also happen to me.
```
import getAssociatedTokenAddress
Module '"@solana/spl-token"' has no exported member 'getAssociatedTokenAddress'.ts(2305)
```
after cmd click the source, it read from spl-token used by metaplex
```
/node_modules/@metaplex/js/node_modules/@solana/spl-token/package.json
```
on that file i saw the version is
```
  "name": "@solana/spl-token",
  "version": "0.1.8",
```
i have `"@solana/spl-token": "^0.2.0"` on my package.json tho.

i used it on my anchor test file and it still works. its just the ts lint reading from `@solana/spl-token@0.1.8` of metaplex instead of `@solana/spl-token@0.2.0`

im not sure if my ts config has dependency resolving failure, but i quite agree that updating spl-token on metaplex would solve this
